content/ember/v1/ember-reduce-computed-property.md

```diff
@@ -1,10 +1,12 @@
 ---
-id: ember-reducecomputed-ember-arraycomputed
+id: ember-reducecomputedproperty-ember-arraycomputedproperty
 title: Ember.ReduceComputedProperty / Ember.ArrayComputedProperty
 until: ''
 since: '1.13'
 ---
 
+<!-- missing Ember.ReduceComputed / Ember.ArrayComputed see ember-reduce-computed.md -->
```

Comment:
i set the `id` manually. but i couldn't get it to appear after `Ember.ReduceComputed / Ember.ArrayComputed`. will check if the id's are alphabetical.

what is the meaning / naming convention of the id?